asllib/doc/ASLTypingReference.tex

```diff
@@ -1406,10 +2190,21 @@ \section{TypingRule.EGlobalVarConstantVal \label{sec:TypingRule.EGlobalVarConsta
 
 \begin{emptyformal}
   \subsection{Formally}
+\[
+\inferrule{
+  G.\constantsvalues \entails x : V\\
+  G.\storagetypes \entails x : (T, \texttt{GDK\_Constant}))\\
+  }
+{(G,L) \entails \evar(x) : (T, \eliteral{V})}
+\]
 \end{emptyformal}
 
 \isempty{\subsection{Comments}}
 
+\todocomment{
```

Comment:
please add as part of this PR

Reply:
This may be a bug in the code - I think we can drop this case as there's the case of TypingRule.EGlobalVar. Check with @HadrienRenaud